source/catalog.html

```diff
@@ -0,0 +1,197 @@
+      </div>
+    </header>
+
+    <main class="page-main">
+      <section class="contest">
+        <h2 class="visually-hidden">название конкурса</h2>
+        <div class="contest__wrapper">
+          <p class="contest__excerpt">Конкурс на самое безбашенное путешествие с Пинк</p>
+        </div>
+      </section>
+
+      <section class="best-story">
+        <h2 class="visually-hidden">личные данные</h2>
+        <div class="best-story__wrapper">
+          <p class="contest__excerpt">Поделитесь своей историей и получите шанс выиграть ценный приз — 1000 миль на вашу бонусную карту!</p>
+          <form class="best-story__list" action="index.html" method="post">
+            <div class="best-story__item">
+              <label for="familiya">ФАМИЛИЯ:</label>
+              <input type="text" name="familiya" id="familiya" value="Укажите фамилию">
+            </div>
+            <div class="best-story__item">
+              <label for="name">ИМЯ:</label>
+              <input type="text" name="name" id="name" value="Введите ваше имя">
+            </div>
+            <div class="best-story__item">
+              <label for="patronymic">ОТЧЕСТВО:</label>
+              <input type="text" name="patronymic" id="patronymic" value="Ну и отчество тоже">
+            </div>
+          </form>
+        </div>
+      </section>
+
+      <section class="travel-achievements">
+        <h2 class="visually-hidden">ваши достижения</h2>
+        <div class="travel-achievements__wrapper">
+          <p class="travel-achievements__excerpt">Ваши безбашенные достижения в путешествии</p>
+          <form class="travel-achievements__list" action="index.html" method="post">
+            <div class="travel-achievements__item">
+              <input type="checkbox" name="selfie" id="selfie" checked>
+              <label for="selfie">Сделал селфи с акулой</label>
+            </div>
+            <div class="travel-achievements__item">
+              <input type="checkbox" name="beach" id="beach" checked>
+              <label for="beach">Обгорел на пляже</label>
+            </div>
+            <div class="travel-achievements__item">
+              <input type="checkbox" name="star" id="star">
+              <label for="star">Видел Чака Норриса</label>
+            </div>
+            <div class="travel-achievements__item">
+              <input type="checkbox" name="shopping" id="shopping">
+              <label for="shopping">Накупил сувениров</label>
+            </div>
+            <div class="travel-achievements__item">
+              <input type="checkbox" name="tower" id="tower" checked>
+              <label for="tower">Удержал башню</label>
+            </div>
+            <div class="travel-achievements__item">
+              <input type="checkbox" name="hotel" id="hotel">
+              <label for="hotel">Разгромил отель</label>
+            </div>
+          </form>
+        </div>
+      </section>
+
+      <section class="Contact-details">
```

Comment:
```suggestion
      <section class="contact-details">
```
Здесь и далее - не должно быть заглавных букв в названиях классов.